sw/host/opentitantool/src/main.rs

```diff
@@ -45,6 +53,9 @@ struct Opts {
     #[structopt(long, default_value = "off")]
     logging: LevelFilter,
 
+    #[structopt(long, help = "Parse and execute the argument as a command")]
+    exec: Vec<String>,
```

Comment:
How is it that you tell structopt to expect `--exec "a" --exec "b"`, rather than `--exec "a" "b"` ?  I seem to remember the default being the latter, when dealing with the `--conf` option, it causing `--conf a.json bootstrap` to be parsed as two file names, rather than a file name and a command verb.

Reply:
This is not well documented by structopt.  You need to use the `--exec="stuff"` form to get the desired behavior.  You are correct that `--exec "a" bootstrap` will get parsed into `exec=["a", "bootstrap"]`, whereas `--exec="a" bootstrap` will get parsed into `exec=["a"], command=BootStrapCommand()`.

I'll take a closer look at structopt and perhaps file an issue with them.

Reply:
After a bit more study of the structopt and clap documentation, I think we should add `number_of_values(1)` to any multi-args.  This forces the flag to be satisfied after consuming one argument without requiring the `--foo=bar` form, but allows flags which take a Vec<T> to be specified multiple times to supply additional values into the Vec.

See https://docs.rs/clap/2.34.0/clap/struct.Arg.html#method.number_of_values